python/cudf/cudf/core/column/lists.py

```diff
@@ -178,6 +178,40 @@ def __init__(self, column, parent=None):
             )
         super().__init__(column=column, parent=parent)
 
+    def get(self, index):
+        """
+        Extract element at the given index from each component
+
+        Extract element from lists, tuples, or strings in
+        each element in the Series/Index.
+
+        Parameters
+        ----------
+        index : int
+
+        Returns
+        -------
+        Series or Index
+
+        Examples
+        --------
+        >>> s = cudf.Series([[1, 2, 3], [3, 4, 5], [4, 5, 6]])
+        >>> s.list.get(-1)
+        0    3
+        1    5
+        2    6
+        dtype: int64
+        """
+        min_col_list_len = self.len().min()
+        if abs(index) > min_col_list_len:
+            raise IndexError("list index out of range")
+        if (index >= 0 and index < min_col_list_len) or (
+            index < 0 and abs(index) <= min_col_list_len
+        ):
```

Comment:
`-min_col_list_len <= index < min_col_list_len` ?

To put in words: the get index is contrained by the shortest list of the column. By cudf convention, index range is `[-len, len)`.